src/main/java/core/basesyntax/service/impl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+
+    @Override
+    public List<String> read(String readPath) {
+        Path path = Paths.get(readPath);
+        List<String> lines;
+
+        try {
+            lines = Files.readAllLines(path);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file " + readPath);
```

Comment:
```suggestion
            throw new RuntimeException("Can't read from file " + readPath, e);
```
варто передавати вихідний об'єкт помилки